mbtiles/src/mbtiles.rs

```diff
@@ -39,6 +40,82 @@ pub struct Metadata {
     pub json: Option<JSONValue>,
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct LevelDetail {
+    pub zoom: u8,
+    pub count: u64,
+    pub smallest: u64,
+    pub largest: u64,
+    pub average: f64,
+    pub bbox: Bounds,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct Statistics {
+    pub file_path: String,
+    pub file_size: u64,
+    pub schema: MbtType,
+    pub page_size: u64,
+    pub level_details: Vec<LevelDetail>,
+    pub count: u64,
+    pub smallest: Option<u64>,
+    pub largest: Option<u64>,
+    pub average: f64,
+}
+
+impl Display for Statistics {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        writeln!(f, "File: {}", self.file_path).unwrap();
+        writeln!(
+            f,
+            "FileSize: {:.2}B",
+            SizeFormatterBinary::new(self.file_size)
+        )
+        .unwrap();
```

Comment:
This style makes it a bit more readable. Same for others below
```suggestion
        let file_size = SizeFormatterBinary::new(self.file_size);
        writeln!(f, "FileSize: {file_size:.2}B").unwrap();
```